modules/monitoring-understanding-the-cluster-observability-operator.adoc

```diff
@@ -0,0 +1,32 @@
+//Module included in the following assemblies:
+//
+// monitoring/cluster_observability_operator/cluster-observability-operator-overview.adoc
+
+:_content-type: CONCEPT
+[id="understanding-the-cluster-observability-operator_{context}"]
+= Understanding the Cluster Observability Operator
+
+A default monitoring stack created by the Cluster Observability Operator (COO) includes a highly available Prometheus instance capable of sending metrics to an external endpoint by using remote write.
+
+Each COO stack also includes an optional Thanos Querier component, which you can use to query a highly available Prometheus instance from a central location, and an optional Alertmanager component, which you can use to set up alert configurations for different services.
+
+[id="advantages-of-using-cluster-observability-operator_{context}"]
+== Advantages of using Cluster Observability Operator
+
+The `MonitoringStack` CRD used by the COO offers an opinionated default monitoring configuration for COO-deployed monitoring components, but you can customize it to suit more complex requirements.
+
+Deploying a COO-managed monitoring stack can help meet monitoring needs that are difficult or impossible to address by using the core platform monitoring stack deployed by the Cluster Monitoring Operator (CMO).
+A monitoring stack deployed using COO has the following advantages over core platform and user workload monitoring:
+
+* *Extendability*. Users can add more metrics to a COO-deployed monitoring stack, which is not possible with core platform monitoring without losing support.
```

Comment:
Maybe colons after the bolded bits (instead of periods), for each of these? Alternatively, [description lists](https://docs.asciidoctor.org/asciidoc/latest/lists/description/#basic-description-list) are fun (and then you don't need any punctuation).

Reply:
Good idea to use a description list. Updated per your suggestion. Thanks!